src/components/Caption.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Caption() {
```

Comment:
Well, logically, the component should be named 'Card' since the caption identity is more like a card-header. CSS classes should be renamed correspondingly.